### ISSUE TITLE
[Question] Missing creatures

### ISSUE THREAD

**Issue by Pwntus** [orts/server#211]
Is there a reason why so many **Oramond** creatures are missing? I'm almost done with the Seacrest Serpent but it's really hard to find the right ID's. Should I wait until an updated OTB file appears?

![Image of Seacrest Serpent](http://i.imgur.com/z0dzCPn.png)

**Comment by Schiffers** [comment]
what id ? of outfit?

**Comment by EPuncker** [comment]
yes, the reason is that no one (AFAIK) is working on them xD (or is willing to)

**Comment by EPuncker** [comment]
@Schiffers that is jersey outfit
https://github.com/otland/forgottenserver/blob/master/data/XML/outfits.xml

**Comment by WibbenZ** [comment]
Give me links to wikia with the outfits and ill gice you the id.

**Comment by EPuncker** [comment]
@WibbenZ I guess he was asking about the jersey outfit in the pic :P

**Comment by WibbenZ** [comment]
Oh tought me ment the new sea serpent~

**Comment by Schiffers** [comment]
well, is about the serpent, cuz is "oramond creature" ...
but the ID is: 
seacrest serpent - 675
deep horror - 676
the yellow monster that I forget the name "horror something" - 677
professor maxxen - 679
glooth bomb - 680
deep terror tentacle - 681
glooth glider - 682

**Comment by Pwntus** [comment]
@Schiffers Thanks, have you got the IDs for the corpses also?
I also found a bug in creature "Blood Beast", when it is killed it becomes a "slain Cyclops".

**Comment by Schiffers** [comment]
bro, the case about the blood beast, is just change the ID in the blood beats.xml in monsters ....
of ID's of corpses I will try find the ids of the monsters that I post here..
seacrest serpent - from: 24262 to: 24265.
deep terror - from: 24269 to: 24271.
the yellow monster that I forget the name "horror something" - from: 24276 to: 24278.
professor maxxen - from: 24279 to:24281.
blood beats correct - from: 23351 to 23354.
human gree - from: 24251 to: 24254.
human purple - from: 24257 to 24260.

**Comment by EPuncker** [comment]
and please keep in mind that the corpse id on monster file MUST be the one that decays in 10 seconds

**Comment by MVJohnsen** [comment]
Minotaur Amazon, Execowtioner, Moohtant
